server/tserver/src/test/java/org/apache/accumulo/tserver/LargestFirstMemoryManagerTest.java

```diff
@@ -197,14 +196,9 @@ public Instance getInstance() {
   @Test
   public void testDeletedTable() throws Exception {
     final String deletedTableId = "1";
-    Function<String,Boolean> existenceCheck = new Function<String,Boolean>() {
-      @Override
-      public Boolean apply(String tableId) {
-        return !deletedTableId.equals(tableId);
-      }
-    };
+
     LargestFirstMemoryManagerWithExistenceCheck mgr =
-        new LargestFirstMemoryManagerWithExistenceCheck(existenceCheck);
+        new LargestFirstMemoryManagerWithExistenceCheck(tableId -> !deletedTableId.equals(tableId));
```

Comment:
It would be nice if you could do: `!deletedTableId::equals` for boolean functions. Sadly, when you negate the lambda, you have to use the long form.